data/json/recipes/other/materials.json

```diff
@@ -32,8 +32,7 @@
       [ [ "forge", 80 ], [ "char_forge", 80 ] ]
     ],
     "components": [
```

Comment:
You can run the JSON linter on a file via `./tools none data/json/foo.json` to get canonical format